.github/workflows/gen.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Node Client Generator and Create Pull Request
+on:
+  schedule:
+    # Run at the end of every day
+    - cron: 0 0 * * *
+jobs:
+  create_pr:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout this repo
+      uses: actions/checkout@v2
+      with:
+        repository: googleapis/google-cloudevents-nodejs
+        path: google-cloudevents-nodejs
+    - name: Checkout client repo
+      uses: actions/checkout@v2
+      with:
+        repository: googleapis/google-cloudevents
+        path: google-cloudevents
+    - name: Setup Node for gen/postgen scripts
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - name: Run the generator
+      run: |
+        cd google-cloudevents-nodejs
+        npm i
+        npm run build
+        npm run gen
+    - name: Commit and create PR is needed
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        cd google-cloudevents-nodejs
+        # If a generator PR DNE
+        if ! [[ `gh pr list --base main --search "Run the generator"` ]]; then
+          # If there is a non-zero diff after running generator
+          if [[ `git status --porcelain` ]]; then
+            git config --global user.email "yoshi-code-bot@google.com"
```

Comment:
Do we need permission to use this identity?

Reply:
Nope, git config emails don't need permission.

We use this email in other places like here:
https://github.com/googleapis/google-cloudevents/blob/main/.github/workflows/gen_pr.yaml

Reply:
I meant more as a social thing: does the Yoshi team know we're using this? Is it okay for other teams to maintain libraries that use it?

Reply:
In terms of the more social aspect, we've gotten approval for using this as the email address already. I've forwarded an email to you about this just now. I'm happy to kick off another email or meeting if you'd like, but I don't see concern right now.